### DISCUSSION TITLE
Calculating shm-size

### DISCUSSION THREAD

**Discussion by cben0ist** [blakeblackshear/frigate#390] [Ask A Question] [answered]
Hi everyone,
Question about "[Calculating shm-size](https://github.com/blakeblackshear/frigate#calculating-shm-size)"
I now have 8 cameras setup in my frigate and I figured I should worry about that but... where is that setting located ? 
Thanks

**Comment by blakeblackshear** [comment, marked as answer]
It is a parameter in the docker-compose file

**Reply by kinnujan** [reply]
Is it possible to change shm-size when using HassOS -component?

**Comment by cben0ist** [comment]
Found it !
    shm_size: '4g'
Thanks
